testbed/stack.ts

```diff
@@ -0,0 +1,9 @@
+import * as cdk from '@aws-cdk/core'
+import { Testbed, TestbedProps } from './construct';
+
+export class TestbedStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props: TestbedProps) {
+    super(scope, id, props);
+    new Testbed(this, "testbed", props)
```

Comment:
I see a convention of "Testbed" for components elsewhere. Why lowercase "testbed"?